web/landing_page.go

```diff
@@ -106,6 +106,11 @@ func NewLandingPage(c LandingConfig) (*LandingPageHandler, error) {
 }
 
 func (h *LandingPageHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path != "/" {
+		w.Write([]byte("Not Found"))
+		w.WriteHeader(http.StatusNotFound)
```

Comment:
```suggestion
		http.NotFound(w, r)
```

Reply:
Excellent point. I always forget about this helper function existing.